iptables/iptables.go

```diff
@@ -336,7 +335,7 @@ func makeJumpFromChainToAnotherForAllProtocols(
 }
 
 func makeShowAllRules() *exec.Cmd {
-	return exec.Command("iptables-save")
+	return exec.Command("iptables-save", "-t", "nat")
```

Comment:
> I reverted to iptables-save, however I had add -t nat as arguments to iptables-save.
 Otherwise it tries to read the wanted table from /proc/net/ip_tables_names which is only readable by root:

We might be looking at operating on the `mangle` table to add support for tproxy in the coming weeks. Do you anticipate this getting more complicated? How would our ability to add rules in different tables be affected by this change?

Reply:
adding rules is just as easy as `iptabels -t mangle ...` However the `makeShowAllRules` should then also include the output of `iptables-save -t mangle`.

Reply:
Awesome, sounds good!